### PR TITLE
Remove traces of plone.directives.form

### PR DESCRIPTION
**Pull request by gforcada**

### CODE REVIEW

**Comment by davisagli** [comment]
This should be changed to `plone.supermodel.model.Schema` instead of `Interface`; checkers for schema directives are run for subclasses of Schema but not (I think) subclasses of Interface.

**Comment by gforcada** [comment]
I was wondering about that as on docs and code I see a mixture of both Interface and Schema being used.

**Comment by gforcada** [comment]
@davisagli @jone ready to review and merge

File: README.rst [hunk 1]
```diff
@@ -65,7 +65,7 @@ you can mark the field searchable that way::
 
     <model xmlns="http://namespaces.plone.org/supermodel/schema"
            xmlns:indexer="http://namespaces.plone.org/supermodel/indexer">
-      <schema based-on="plone.directives.form.schema.Schema">
+      <schema based-on="from plone.supermodel.model.Schema">
```
Comment (mauritsvanrees) [inline README.rst]: This line looks wrong. Bad search/replace. :)

Comment (gforcada) [inline README.rst]: Oops! :blush:

**Comment by gforcada** [comment]
@mauritsvanrees thanks for the review, that should be fixed :-)

**Comment by jone** [comment]
Looks good to me 👍

**Pull request merged**